src/TransitionSystems/location_tuple.rs

```diff
@@ -71,6 +88,7 @@ impl LocationTuple {
             loc_type,
             left: Some(Box::new(left.clone())),
             right: Some(Box::new(right.clone())),
+            is_partial_location: false,
```

Comment:
What if we merged with a partial? then this would break

Reply:
Fixed